components/21-atoms/table/table-mobile.functions.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+var MobileTable = function (element) { // eslint-disable-line no-unused-vars
+  /**
+   * The table wrapper.
+   */
+  var table = null;
+
+  /**
+   * The table caption.
+   */
+  var caption = null;
+
+  /**
+   * The responsive list version of the table.
+   */
+  var tableList = null;
+
+  /**
+   * Add a wrapper div with class .table-list as next sibling to the table.
+   */
+  var addTableList = function () {
+    var tableListContainer = document.createElement('div');
+    tableListContainer.setAttribute('class', 'table-list');
+    table.parentNode.insertBefore(tableListContainer, tableListContainer.nextSibling);
+
+    tableList = table.nextElementSibling;
+  };
+
+  /**
+   * Adds the individual items to the list <ul></ul> of the table-list wrapper.
```

Comment:
If you're gonna be all fancy with comments and all,
add params ;)

Reply:
👍